copy/manifest_test.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/containers/image/manifest"
 	"github.com/containers/image/types"
 	"github.com/opencontainers/image-spec/specs-go/v1"
+	imgspecv1 "github.com/opencontainers/image-spec/specs-go/v1"
```

Comment:
This is already imported as `v1` just above, please use that one. Or rename the package import if you must, but import the package only once.

Reply:
fixed